taca/analysis/analysis.py

```diff
@@ -130,8 +132,38 @@ def trigger_analysis(run_id):
                          "of {}. Please check the logfile and make sure to "
                          "start the analysis!".format(os.path.basename(run_id))))
 
+def find_samplesheet(run):
+    """Find the samplesheet for the given run
 
-def prepare_sample_sheet(run):
+    :param taca.illumina.Run run: Run to find the SampleSheet for
+    :returns str: Path to the SampleSheet
+    """
+    if run.run_type == 'MiSeq':
+        return os.path.join(run.run_dir, 'Data', 'Intensities', 'BaseCalls', 'SampleSheet.csv')
+    else:
```

Comment:
@guillermo-carrasco or who evetr wil work on this (maybe @me): I am not a big fun of "if MiSeq do something otherwise something else". There should be a choice for every sequencer type we have, or better an abstract class "run" that is specialised in MiSeq, HiSeqX, HiSeq2500 and so on.. maybe this is what happens next in the code but wnat to write it now as I am flying and this is the only work I can do given the connection :-D